source/includes/_api.md

```diff
@@ -18,6 +17,20 @@ callStats.initialize(AppID, AppSecret, localUserID, csInitCallback, csStatsCallb
 `csStatsCallback`  | Optional | Callback | asynchronously reports the conference statistics.
 `configParams`  | Optional | JSON| it is the set of parameters to enable/disable certain features in the library.
 
+## callstats.initialize() with third party authentication
```

Comment:
initialize(). We are overloading it, the js_api.yml commit was accidental (made before I realized it's no longer in use)